packages/spineClient/src/sandbox-spine-client.ts

```diff
@@ -15,4 +15,9 @@ export class SandboxSpineClient implements SpineClient {
     // This is not implemented as sandbox lambda does not use this code
     throw new Error("INTERACTION_NOT_SUPPORTED_BY_SANDBOX")
   }
+
+  async isCertificateConfigured(): Promise<boolean> {
```

Comment:
no need for async

Reply:
The method has been changed - async has been removed